compiler/qsc_frontend/src/compile/preprocess.rs

```diff
@@ -143,17 +147,31 @@ fn matches_config(attrs: &[Box<Attr>], capabilities: TargetCapabilityFlags) -> b
                         return true; // Unknown capability, so we assume it matches
                     }
                 }
+                ExprKind::UnOp(UnOp::NotL, inner) => {
+                    if let ExprKind::Path(path) = inner.kind.as_ref() {
+                        if let Ok(capability) =
+                            TargetCapabilityFlags::from_str(path.name.name.as_ref())
+                        {
+                            disallowed_capabilities |= capability;
+                        } else {
+                            return true; // Unknown capability, so we assume it matches
+                        }
+                    } else {
+                        return true; // Unknown config attribute, so we assume it matches
+                    }
+                }
                 _ => return true, // Unknown config attribute, so we assume it matches
             }
         } else {
             // Something other than a parenthesized expression, so we assume it matches
             return true;
         }
     }
-    if found_capabilities == TargetCapabilityFlags::empty() {
```

Comment:
Should we update the existing `@Config(Adaptive)` to be `@Config(not Base)`?

Reply:
I don't think so... since `Unrestricted` is a special case meaning "all capabilities" it includes "adaptive." But I could be convinced otherwise if you think it's clearer in the code base.

Reply:
I'll take a look through the standard library and update any markers. I think we did expect that the code could use different attributes once the new RCA check pass was default and base profile check was removed.